sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentModelAdministrationClient.cs

```diff
@@ -114,8 +114,12 @@ public DocumentModelAdministrationClient(Uri endpoint, TokenCredential credentia
         /// <summary>
         /// Build a custom document analysis model from a collection of documents in a blob storage container.
         /// </summary>
-        /// <param name="trainingFilesUri">An externally accessible Azure storage blob container Uri pointing to the container that has your training files.
-        /// For more information see <see href="https://docs.microsoft.com/azure/applied-ai-services/form-recognizer/build-training-data-set">here</see>.</param>
+        /// <param name="trainingFilesUri">
+        /// An externally accessible Azure storage blob container URI pointing to the container that has your training files.
```

Comment:
```suggestion
        /// An externally accessible Azure Blob Storage container URI pointing to the container that has your training files.
```

nit:  Branding.  _([ref](https://docs.microsoft.com/azure/storage/blobs/))_

Reply:
Leaving one comment, but this occurs in the other doc comments as well